### PR TITLE
Include python3 package for vdsm build requirements

### PR DESCRIPTION
**Pull request by bronhaim**

### CODE REVIEW

File: source/develop/developer-guide/vdsm/developers.html.md [hunk 1]
```diff
@@ -41,7 +41,13 @@ Install the following packages before you attempt to build VDSM:
        python-ethtool m2crypto python-dmidecode python-netaddr python-inotify python-argparse git \
        python-cpopen bridge-utils libguestfs-tools-c pyparted openssl libnl3 libtool gettext-devel python-ioprocess \
        policycoreutils-python python-simplejson python-blivet python-six mom ovirt-vmconsole
-
+       
+Install Python3 packages (for fedora)
```
Comment (sandrobonazzola) [inline source/develop/developer-guide/vdsm/developers.html.md]: add a new line here. check how this change is rendered with the view button.
current text needs formatting fixes.

**Comment by bronhaim** [comment]
done

**Comment by bronhaim** [comment]
can you merge that?

**Comment by mykaul** [comment]
I think for Fedora it should be 'dnf install' - but I'll merge it as is for the time being.

**Pull request merged**